lib/LedgerSMB/Entity/Note.pm

```diff
@@ -8,7 +8,7 @@ employees, etc.
   @notes = LedgerSMB::Entity::Bank->list($entity_id, [$credit_id]);
   $note->add;
 
-=head1 DESCRIPTION
+=head1 DESCRIPTION 
```

Comment:
Good to know,  there is the comment about reformating code at the end of changes.  And I was looking forward to that; I will diminish my expectations.  I don't think I'm fussy regarding format, but I find long lines difficult  and consistent indentation helps me a lot with finding scopes.  

Can one get stacked snapshots of blame?   Or  is there an easy way to view the commits of a file in reverse order through time?

Is the difficulty you mention just that there is an extra commit?